src/reactive/keystone_saml_mellon_handlers.py

```diff
@@ -13,51 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import uuid
-
 # import to trigger openstack charm metaclass init
-import charm.openstack.keystone_saml_mellon # noqa
+import charm.openstack.keystone_saml_mellon as keystone_saml_mellon  # noqa
 
 import charms_openstack.charm as charm
 import charms.reactive as reactive
-import charms.reactive.flags as flags
-
-import charmhelpers.core.unitdata as unitdata
 
 from charms.reactive.relations import (
```

Comment:
I guess it is a matter of taste, but I prefer this:

    import charms.reactive as reactive

And then reference what you need from the module throughout the code.

Reply:
Done